packages/thirdweb/src/react/web/ui/ConnectWallet/Avatar.tsx

```diff
@@ -0,0 +1,49 @@
+import type React from "react";
+import type { Address } from "viem";
+import type { ThirdwebClient } from "../../../../client/client.js";
+import {
+  useEnsAvatar,
+  useEnsName,
+} from "../../../../react/core/utils/wallet.js";
+import { Blobbie } from "./Blobbie.js";
+
+export type AvatarProps = {
+  client: ThirdwebClient;
+  walletAddress?: Address;
+  style?: React.CSSProperties;
+  className?: string;
+};
```

Comment:
Why is `walletAddress` prop optional?

Reply:
Yeah you are right, maybe I should have done `{address && <Avatar .../>` instead of making the prop optional